client/src/App.js

```diff
@@ -80,7 +80,7 @@ function App() {
         <footer className="bg-gray-800 w-full bottom-0 items-center flex flex-full flex-col py-20">
           <address class="flex space-x-10">
             <div>
-              <a href="#">
+              <a href="https://twitter.com/?lang=en">
```

Comment:
It looks like maybe you didn't commit before checking out another branch do the browser tab story? I'm wondering if that's why these changes are here too. Not a big deal, but if you can keep these changes in the other PR that'd be great!